src/common/user-context.tsx

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
 
 interface UserContextInterface {
+  currentUser: LoginResponse
   unfilteredUsers: DummyUser[]
   filteredUsers: DummyUser[]
   updateUnfilteredUsers: React.Dispatch<React.SetStateAction<DummyUser[]>>
   updateFilteredUsers: React.Dispatch<React.SetStateAction<DummyUser[]>>
+  updateCurrentUser: React.Dispatch<React.SetStateAction<LoginResponse>>
 }
 
 const value = {
+  currentUser: {} as LoginResponse,
   unfilteredUsers: [] as DummyUser[],
   filteredUsers: [] as DummyUser[],
```

Comment:
Same thing here :)
Although for `DummyUser[]` I'm pretty sure the empty array satisfies the type, so you can just leave `filteredUsers: DummyUser[]` in line 6 as is and remove the assertions here.